packages/babel-core/src/config/files/configuration.js

```diff
@@ -18,26 +18,22 @@ import type { CallerMetadata } from "../validation/options";
 
 const debug = buildDebug("babel:config:loading:files:configuration");
 
-const BABEL_CONFIG_JS_FILENAME = "babel.config.js";
-const BABEL_CONFIG_JSON_FILENAME = "babel.config.json";
 const ROOT_CONFIG_FILENAMES = [
-  BABEL_CONFIG_JS_FILENAME,
-  BABEL_CONFIG_JSON_FILENAME,
+  "babel.config.js",
```

Comment:
What about `.mjs`? Would that be an issue in the future?

Reply:
We don't support configurations which must be loaded asynchronously (like es modules) yet.